tests/test_ec2/test_route_tables.py

```diff
@@ -147,6 +147,22 @@ def test_route_tables_filters_standard():
         [route["GatewayId"] == igw.id for table in resp for route in table["Routes"]]
     )
 
+    # Filter by route destination CIDR block
+    resp = client.describe_route_tables(
+        Filters=[
+            {"Name": "route.destination-cidr-block", "Values": ["10.0.0.4/24"]},
+        ]
+    )["RouteTables"]
+    resp.should.have.length_of(1)
```

Comment:
The EC2 tests run in parallel, so there can be multiple route tables in the system that match this query.

Reply:
Fixed with c9be51fcf273507baecb7a7d5e5fd244d73452fe